modules/rhel-adding-node.adoc

```diff
@@ -0,0 +1,63 @@
+// Module included in the following assemblies:
+//
+// * machine_management/adding-rhel-compute.adoc
+
+[id="rhel-adding-node-{context}"]
+= Adding a RHEL compute machine to your cluster
+
+You can add compute machines that use Red Hat Enterprise Linux as the operating
+system to a {product-title} {product-version} cluster.
+
+.Prerequisites
+
+* You installed the required packages and performed the necessary configuration
+on the machine that you run the playbook on.
+* You prepared the RHEL hosts for installation.
+
+.Procedure
+
+Perform the following steps on the machine that you prepared to run the
+playbook:
+
+. Extract the pull secret for your cluster.
++
+----
+$ oc -n openshift-config get -o jsonpath='{.data.\.dockerconfigjson}' secret pull-secret | base64 -d | jq .
+----
+
+. Save the pull secret in a file that is named `pull-secret.txt`.
+
+. Create an `openshift-ansible/inventory/hosts` file that defines your compute
```

Comment:
just create an inventory file rather than create an openshift-ansible/inventory/hosts file, 
as normal user do not have create file right in /usr/share/ansible/openshift-ansible folder by default.

for run playbook,  recommend this
$ ansible-playbook -i /path/to/inventory playbooks/scaleup.yml